po/uk.po

```diff
@@ -3165,34 +3165,35 @@ msgstr "неприпустиме посилання: \"%s\""
 
 #: builtin/bisect.c:897
 msgid "You need to start by \"git bisect start\"\n"
-msgstr ""
+msgstr "Вам потрібно почати з “git bisect start”\n"
```

Comment:
```suggestion
msgstr "Вам потрібно почати з \"git bisect start\"\n"
```

Reply:
I hope I've fixed the "smart" quotes issue.